src/error.rs

```diff
@@ -21,67 +23,89 @@ pub(crate) enum Error {
   AddressResolutionIo {
     input: String,
     source: io::Error,
+    #[cfg(not(test))]
```

Comment:
At first I thought we could just conditionally switch backtrace printing on and off. But for most tests, backtraces are not printed through `Error::print_backtrace`, but e.g. by calling `unwrap()` on an error that contains a backtrace, which will then be printed out through the `Debug` impl. This PR just removes the `backtrace` field when compiling tests.